src/app-layout/__integ__/app-layout-focus-delegation.test.ts

```diff
@@ -240,9 +258,32 @@ describe.each(['classic', 'visual-refresh', 'visual-refresh-toolbar'] as const)(
         )
       );
 
+      test(
+        'split panel focus toggles between open and close buttons',
+        setupTest(
+          async page => {
+            //Altermatomg between triggers because test-1 trigger hidden in overflow menu on mobile,
```

Comment:
I usually don't mind typos in comments but here "Altermatomg" is something else :)